src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -110,8 +110,8 @@ class ResultsList extends Component {
                 <SearchResult result={r} />
               </div>
             ) : (
-              <SearchResult key={r.id} result={r} />
-            );
+                <SearchResult key={r.id} result={r} />
+              );
             /* eslint-enable prettier/prettier */
           })}
         </div>
```

Comment:
@aubreyarcangel, I think you forgot to switch back to the master branch when you were switching to this [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/10161). This commit should be reverted.